app/manifest.json

```diff
@@ -12,5 +12,12 @@
   ],
   "background": {
     "service_worker": "/src/01_frameworks-driver.layer/background/background.ts"
-  }
+  },
+  "options_page": "options.html",
+  "web_accessible_resources": [
+    {
+      "resources": ["assets/*"],
```

Comment:
Note: I'm not an expert here, but for some reason the JS wouldn't be loaded by Chrome without this.